src/objects/entities/Entity.ts

```diff
@@ -24,14 +24,14 @@ export type EntityConfigType = {
   scale: number;
   // eslint-disable-next-line @typescript-eslint/ban-types
   collideCallback?: Function;
-  collisionCategory?: CollisionCategories;
+  collisionCategory?: CC;
   craftpixOffset: {
     x: number;
     y: number;
   };
 };
 
-const defaultConfig: EntityConfigType = {
+const defaultConfig = {
```

Comment:
Why are we removing the type here?

Reply:
interestingly, the default values don't match this type because the `| undefined`s are actually defined in the default values... :thinking: